fdbserver/Status.actor.cpp

```diff
@@ -1843,6 +1845,7 @@ ACTOR static Future<JsonBuilderObject> workloadStatusFetcher(Reference<AsyncVar<
 		JsonBuilderObject transactions;
 		transactions["conflicted"] = txnConflicts.getStatus();
 		transactions["started"] = txnStartOut.getStatus();
+		transactions["rejected_for_too_long_queued"] = txnRejectedForTooLongQueued.getStatus();
```

Comment:
Since you modify the status json. I'd expect `*Status` tests to fail--you might need to run more tests. You'll need to modify `mr-status-json-schemas.rst.inc` and `fdbclient/Schemas.cpp` as well.

Reply:
Verified the following tests failed before adding keys in the schema(without the second commit in this PR), with the second commit they no longer have `SevError` 
```
./fast/SidebandWithStatus.toml
./slow/DDBalanceAndRemoveStatus.toml
./rare/LargeApiCorrectnessStatus.toml
```

Reply:
btw, looks like even though they have `SevError` when not changing the schema as below, tests are still considered passed, not sure why.. (`SevError` is gone after changing the schema),.
```
4 test clients passed; 0 test clients failed
Run test:ApiCorrectnessTest Done.

1 tests passed; 0 tests failed.
Waiting for DD to end...

Unseed: 6188
Elapsed: 1345.231831 simsec, 71.331345 real seconds
14 SevError events logged
```

Reply:
I think running with `Joshua`, such a test result will be reported as failure, because `Joshua` checks if there are `SevError` events.